Changelog.md

```diff
@@ -1,8 +1,10 @@
 # Next
 
-## Fixed
-- Fixes Alamofire validation not being performed on `.uploadMultipart` requests.
+### Fixed
+- Fixed Alamofire validation not being performed on `.uploadMultipart` requests.
 [#1591](https://github.com/Moya/Moya/pull/1591) by [@SD10](https://github.com/SD10).
+- Fixed `Response` validation not being performed on stubbed requests.
```

Comment:
can we phrase it the same as the entry above? it is basically the same thing, just fixed for different parts of the library, but might seem like something different

Reply:
I didn't mention Alamofire because technically it doesn't have anything to do with Alamofire 😂 They don't need to know that though